pkgs/shells/bash/5.nix

```diff
@@ -49,7 +49,7 @@ lib.warnIf (withDocs != null)
     # bionic libc is super weird and has issues with fortify outside of its own libc, check this comment:
     # https://github.com/NixOS/nixpkgs/pull/192630#discussion_r978985593
     # or you can check libc/include/sys/cdefs.h in bionic source code
-    ++ lib.optional (stdenv.hostPlatform.libc == "bionic") "fortify";
+    ++ lib.optionals (stdenv.hostPlatform.libc == "bionic") [ "fortify" ];
```

Comment:
It might be better not to change optional for optionals if more than 1 element isn't needed

Reply:
@fabianhjr Can you please reconsider this?

Reply:
Sure, given those reasons and that I don't feel strongly about optional vs optionals I agree with the change